book-store/src/redux/books/books.js

```diff
@@ -0,0 +1,31 @@
+const ADD_BOOK = 'bookStore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+
+const initialState = [];
+
+const addBook = (payload) => ({
+  type: ADD_BOOK,
+  payload,
+});
+
+const removeBook = (payload) => ({
+  type: REMOVE_BOOK,
+  payload,
```

Comment:
* Kindly implement the remove book functionality too as it is a requirement for this project

Reply:
Fixed